pom.xml

```diff
@@ -261,9 +261,8 @@
         <version>4.0</version>
       </dependency>
       <dependency>
-        <groupId>org.jenkins-ci.plugins</groupId>
-        <artifactId>aws-java-sdk</artifactId>
-        <version>1.12.70</version>
+        <groupId>org.jenkins-ci.plugins.aws-java-sdk</groupId>
+        <artifactId>aws-java-sdk-minimal</artifactId>
```

Comment:
Maybe I am missing something. Why is this entry, without a version, even here in `dependencyManagement`? It has no effect on `Plugin-Dependencies`, which lists only `aws-credentials` and `aws-global-configuration`, so we are picking up the actual SDK plugin dep only transitively, right?

```
[INFO] io.jenkins.plugins:artifact-manager-s3:hpi:1.16-SNAPSHOT
[INFO] +- io.jenkins.plugins:aws-global-configuration:jar:1.7:compile
[INFO] |  \- org.jenkins-ci.plugins.aws-java-sdk:aws-java-sdk-minimal:jar:1.12.70:compile
[INFO] |     +- com.amazonaws:aws-java-sdk-core:jar:1.12.70:compile
[INFO] |     |  +- software.amazon.ion:ion-java:jar:1.0.2:compile
[INFO] |     |  +- com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:jar:2.12.3:compile
[INFO] |     |  \- joda-time:joda-time:jar:2.10.12:compile
[INFO] |     +- com.amazonaws:aws-java-sdk-s3:jar:1.12.70:compile
[INFO] |     |  +- com.amazonaws:aws-java-sdk-kms:jar:1.12.70:compile
[INFO] |     |  \- com.amazonaws:jmespath-java:jar:1.12.70:compile
[INFO] |     +- com.amazonaws:aws-java-sdk-sts:jar:1.12.70:compile
```

And why is S3 a dep of the “minimal” plugin? This is a specific service. I would have expected it to be bundled in some `aws-java-sdk-s3` plugin.

Reply:
See https://github.com/jenkinsci/aws-java-sdk-plugin/pull/540 for context.

Reply:
#239